oe_corporate_blocks.post_update.php

```diff
@@ -78,7 +77,6 @@ function _oe_corporate_blocks_import_corporate_links(): void {
 
   foreach ($configs as $config) {
     $config_storage->write($config, $source->read($config));
```

Comment:
Use this instead:
```suggestion
   $config_factory->getEditable($config)->setData($source->read($config))->save();
```

Reply:
Unfortunately, it leads to fatal error in component and in EWCMS to a fatal error:
![Screenshot from 2020-11-27 17-49-58](https://user-images.githubusercontent.com/9432036/100466078-0089cf00-30d9-11eb-9589-e4d6f4e96c25.png)